src/sql/magic.py

```diff
@@ -95,38 +96,13 @@ class SqlMagic(Magics, Configurable):
 
     Provides the %%sql magic."""
 
-    displaycon = Bool(
-        default_value=True, config=True, help="Show connection string after execution"
-    )
+    autocommit = Bool(default_value=True, config=True, help="Set autocommit mode")
```

Comment:
looks like you ordered them alphabetically (?)

Reply:
Yes, I ordered both the code and the documentation for uniformity